runtime/vm/xr64/unsafeHelper.s

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2018, 2018 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+	.text
+# Prototype: void unsafePut64(I_64 *address, I_64 value);
+# Defined in: #Args: 2
+	.align	2
+	.global	unsafePut64
+	.type	unsafePut64, %function
+unsafePut64:
+	str	x1, [x0]
+	ret
+	.size	unsafePut64, .-unsafePut64
+# Prototype: void unsafePut32(I_32 *address, I_32 value);
```

Comment:
It's a fairly minor thing but can you add a blank line between the different functions being defined in this file?  It makes it little easier to pick out where one ends and the next begins

Reply:
Sure.